src/javascript/app/base/client.js

```diff
@@ -108,6 +109,10 @@ const Client = (() => {
         });
     };
 
+    const hideUpgradePulser = () => {
+        $('#topbar-msg').children('a').setVisibility(!/maltainvest/.test(window.location.href));
```

Comment:
this only works for financial account opening page. I think a more general solution would be better. I suggest on each account opening page we hide the pulser for that page individually, then we don't have to worry about url paths changing either